scripts/install/ceres_install.bash

```diff
@@ -2,6 +2,7 @@
 set -e  # exit on first error
 UBUNTU_VERSION=`lsb_release --release | cut -f2`
 SRC_PREFIX_PATH="/usr/local/src/"
```

Comment:
I guess you mean the line `git clone` will fail? Yes.. I will fix it for issue #14.